src/doc/DocDrawingDialog.js

```diff
@@ -56,9 +56,14 @@ class DocDrawingDialog extends AnnotationDialog {
      *
      * @override
      * @protected
+     * @param {Annotation} annotation Annotation to add
      * @return {void}
      */
-    addAnnotation() {}
+    addAnnotation(annotation) {
+        if (!this.element) {
```

Comment:
Shouldn't this also add an annotation if `this.element` already exists?

Reply:
This method only adds the annotation element to the dialog. So since we're not adding comments/multiple annotations for drawings, we don't really need to do much here.

Reply:
Good enough for me :D